app/models/channel.js

```diff
@@ -8,6 +8,9 @@ export default Ember.Object.extend({
   connected: false,
   sockethubChannelId: null,
   topic: null,
+  unreadMessages: false,
+  unreadMentions: false,
+  visible: false, // Current/active channel
```

Comment:
I thought about it and at the time decided that "visible" was more apt. Thinking about it again, it's all the same to me. I just didn't like how "active" made it sound like other channels were somehow inactive, when this very feature is about indicating channel activity in channels that are not currently open/visible.